drift-transport-netty/src/main/java/com/facebook/drift/transport/netty/client/DriftNettyClientConfig.java

```diff
@@ -225,4 +231,55 @@ public DriftNettyClientConfig setCiphers(String ciphers)
                 .splitToList(requireNonNull(ciphers, "ciphers is null"));
         return this;
     }
+
+    public Boolean getConnectionPoolEnabled()
+    {
+        return connectionPoolEnabled;
+    }
+
+    @Config("thrift.client.connection-pool.enabled")
```

Comment:
curious: are the configs defined in this `DriftNettyClientConfig` can be overridden by the client prefix? e.g. `<prefix>.thrift.client.connection-pool.enabled`?

Reply:
Yup